Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
             agent {
                 dockerfile {
                     filename 'Dockerfile.build'
-                    args '--userns=host -v /home/ci/.npm:/.npm'
+                    args '--userns=host --security-opt seccomp:unconfined -v /home/ci/.npm:/.npm'
```

Comment:
`args` property is applied when a container is run i.e during the the ```docker run ...``` command 
Since you are using a dockerfile to build you image you need to add the `--security-opt` option during the building of the image.
That can be achieved using the `additionalBuildArgs` argument.
Like this 
```groovy
agent {
    // Equivalent to "docker build -f Dockerfile.build --build-arg version=1.0.2 ./build/
    dockerfile {
        filename 'Dockerfile.build'
        additionalBuildArgs  '--security-opt seccomp:unconfined'
        args '--userns=host -v /home/ci/.npm:/.npm'
    }
}
```

Reply:
Did that. Failed now with:

+ docker build -t c4e8b0651f8e0a86ae00f97fd483c34233cb61bd --security-opt seccomp:unconfined -f Dockerfile.build .
Error response from daemon: The daemon on this platform does not support setting security options on build

Reply:
I locked the node version to node:18-bullseye-slim and it helped

Reply:
resolving as fixed by reuseNode: true